tests/unit/amo/pages/TestCategory.js

```diff
@@ -110,6 +111,51 @@ describe(__filename, () => {
     );
   }
 
+  const renderWithCategory = ({ category = fakeCategory, props = {} }) => {
+    _categoriesLoad({ results: [category] });
+
+    return render(props, {
+      autoDispatchCategories: false,
+      paramOverrides: {
+        slug: category.slug,
+        visibleAddonType: getVisibleAddonType(category.type),
+      },
+    });
+  };
+
+  const renderFirefoxTheme = (props = {}) => {
+    const category = {
+      ...fakeCategory,
+      application: CLIENT_APP_FIREFOX,
+      type: ADDON_TYPE_THEME,
+    };
+
+    return renderWithCategory({ category, props });
+  };
+
+  const renderAndroidTheme = (props = {}) => {
+    const category = {
+      ...fakeCategory,
+      // This should really be et to CLIENT_APP_ANDROID *but* the reducer
+      // replaces android "theme" categories with firefox categories here:
+      // https://github.com/mozilla/addons-frontend/blob/0ebf7b7fc559dee2cf704bad04905c8c8e430905/src/core/reducers/categories.js#L72-L82
```

Comment:
sorry for my ignorance but what does "et" mean here?

Reply:
nothing 😭 that should be "set"

Reply:
lol.